### ISSUE TITLE
android.versionCode - should be integer

### ISSUE THREAD

**Issue by jaulz** [byCedric/semantic-release-expo#42]
### Description of the bug
When I try to build the application via Expo, I see this error:
```
Field: android.versionCode - should be integer.
```

### To Reproduce
Unfortunately, I cannot tell in detail now.

#### Expected behavior
The version should be dumped as integer.

#### Actual behavior
The version is dumped as a string:
![image](https://user-images.githubusercontent.com/5358638/45622903-e6c6b900-ba85-11e8-9de3-160ea6471dec.png)

### Additional context
Might be cause by upgrade to Expo v30? Or maybe from the new template mechanism? But actually I only see tests where the version code is supposed to be an integer.

**Comment by byCedric** [comment]
Thanks for reporting! I think you are right, might be something related to either version bumpers or the template mechanism. I'll take a look tonight and keep this thread updated!

**Comment by byCedric** [comment]
I think it's related to [this line](https://github.com/byCedric/semantic-release-expo/blob/develop/src/types.ts#L76), sorry for the inconvenience! I'll make a fix when I'm finished at work.

**Comment by byCedric** [comment]
Ok, so it was not related to the line I pointed earlier. This issue is caused by lodash's template function, always returning a string. It also wasn't detected by the tests because I mocked the return value to a numeric value. 

The two lines I edited in the pull requests are forcing the android version code to an integer, as the spec describes. 
> A positive integer used as an internal version number.

It also mocks the return value of `calculateAndroidVersion` as string, and checks if a number is written tot he manifest. I'll release this now! Again, @jaulz thanks for reporting!
